### PR TITLE
Include cdn broker iam path

### PR DESCRIPTION
**Pull request by pburkholder**
Reviewing #275 I noticed the example creds were missing these.

### CODE REVIEW

File: ci/credentials.yml.example [hunk 1]
```diff
@@ -95,12 +95,14 @@ aws_external_s3_tfstate_bucket: EXTERNAL_TERRAFORM_BUCKET
 cdn_broker_username_staging: cdn-broker-staging
 cdn_broker_bucket_staging: cdn-broker-le-verify-staging
 cdn_broker_cloudfront_prefix_staging: cg-staging/*
-cdn_broker_hosted_zone_staging:
+cdn_broker_hosted_zone_staging: ZZZZZZZZZZZZZZ
```
Comment (jmcarp) [inline ci/credentials.yml.example]: Is `ZZZZZZZZZZZZZZ` more descriptive than the empty string?

Comment (pburkholder) [inline ci/credentials.yml.example]: Probably not, so changed to ZONEIDSTRING01, which might be.

Comment (jmcarp) [inline ci/credentials.yml.example]: As an aside, this is why I think example credentials are rarely valuable--our variable names should be descriptive enough in most cases, and if they're not, better to rename than add examples.

Comment (pburkholder) [inline ci/credentials.yml.example]: I have nothing vested in the color of this 🚲  🏚 , but I thought it marginally useful to hint that the value would be a zone id starting with 'Z'. 

Want to merge as is? Or do you prefer an empty string?

**Pull request merged**